test/integration/account_users_test.go

```diff
@@ -4,12 +4,15 @@ import (
 	"context"
 	"testing"
 
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/linode/linodego"
 	. "github.com/linode/linodego"
 )
 
 const usernamePrefix = "linodegotest-"
 
+var ignoreUserTimestampes = cmpopts.IgnoreFields(linodego.User{}, "PasswordCreated")
```

Comment:
What does this var do?

Reply:
I meant to flag that we ignore comparing `password_created`, because it can be nullable. I found similar usage in our code to ignore comparing the timestamp.